Sources/Helpers/OAUtilities.m

```diff
@@ -1897,11 +1897,11 @@ + (CGFloat) getLeftMargin
 + (CGFloat) calculateScreenWidth
 {
     if (NSThread.isMainThread)
-        return UIScreen.mainScreen.bounds.size.width;
+        return [UIApplication sharedApplication].mainWindow.frame.size.width;
```

Comment:
In the CarPlay simulator, [UIApplication sharedApplication].mainWindow can be nil. You need to check if there won't be a crash